packages/server/src/environment.js

```diff
@@ -13,10 +13,7 @@ function isJest() {
 }
 
 function isDev() {
-  return (
-    process.env.NODE_ENV !== "production" &&
-    process.env.BUDIBASE_ENVIRONMENT !== "production"
```

Comment:
As far as I can tell this is our only usage of `BUDIBASE_ENVIRONMENT`. Removing this from the `isDev` check so that we can use this flag to differentiate between prod, preprod and release

Reply:
This enabled tools monitoring tools like elastic to consume metrics from our preprod env when testing out integrations while not polluting the production dataset

Reply:
Agree with this - its weird having a named env var like this has zero function, good to give it one!